releasenotes/notes/add-ecr-sx-equivalences-5b6fe73ec599d1a4.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    Add a decomposition of an :class:`.CXGate` into :class:`.ECRGate` and 1-qubit Clifford gates
+    (up to a global phase) into the :class:`.EquivalenceLibrary`.
+    Add decompositions of an :class:`.HGate` into :class:`.SXGate` (respectively :class:`.SXdgGate`)
+    and :class:`.SGate` (respectively :class:`.SdgGate`) (up to a global phase) into the :class:`.EquivalenceLibrary`.
```

Comment:
I'd probably call this a "feature" rather than a bugfix (unless we've used bugfix before).  You can also make the two sentences two different points if you like:
```yaml
feature:
  - |
    Added a decomposition of :class:`.CXGate` ...
  - |
    Added a decomposition of :class:`.HGate` ...
```